app/api/reviews/index.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+
+const router = express.Router();
+
+const ProjectReviews = require('../../database/models/projectReviewsModel');
+
+router.post('/', async (req, res) => {
+  try {
+    const reviews = [req.body.likely_to_recommend, req.body.supporter_communication, req.body.provided_value];
+    let correctBool = true;
+    for (let i = 0; i < reviews.length; i += 1) {
+      if (!(reviews[i].isInteger() && reviews[i] >= 1 && reviews[i] <= 5)) {
+        correctBool = false;
+        break;
+      }
+    }
+    if (correctBool) {
+      const projectReview = await ProjectReviews.insert(req.body);
+      res.status(201).json({ projectReview });
+    } else {
+      res.status(400).json({ message: 'Incorrect input for ratings' });
+    }
```

Comment:
Use a [guard clause](https://medium.com/@scadge/if-statements-design-guard-clauses-might-be-all-you-need-67219a1a981a) here.
```javascript
if (!constantBool) return res.status(400).json({ message });

const projectReview = await ProjectReviews.insert(req.body);
return res.status(201).json({ projectReview });
```